packages/typedoc-plugin/src/markdown-theme.ts

```
@@ -0,0 +1,14 @@
+import type { Reflection } from 'typedoc'
+import { MarkdownTheme, type MarkdownPageEvent } from 'typedoc-plugin-markdown'
+
+import { MyMarkdownThemeContext } from './markdown-theme-context'
+
+export class MyMarkdownTheme extends MarkdownTheme {
+  constructor(...args: ConstructorParameters<typeof MarkdownTheme>) {
+    super(...args)
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)